lib/matplotlib/tests/test_pyplot.py

```diff
@@ -206,7 +206,7 @@ def test_subplot_replace_projection():
     ax = plt.subplot(1, 2, 1)
     ax1 = plt.subplot(1, 2, 1)
     ax2 = plt.subplot(1, 2, 2)
-    # This will delete ax / ax1 as they fully overlap
```

Comment:
At least one should be kept and the warning captured.

Reply:
sure